proto-lens/src/Data/ProtoLens/Encoding/Bytes.hs

```diff
@@ -34,6 +44,12 @@ import Foreign.Marshal.Alloc (alloca)
 import Foreign.Storable (Storable, peek, poke)
 import System.IO.Unsafe (unsafePerformIO)
 
+runParser :: Parser a -> ByteString -> Either String a
```

Comment:
Maybe add a bit of haddock for public functions?

Reply:
Done.